app/models/micropost.rb

```diff
@@ -0,0 +1,16 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  validates :user_id, presence: true
+  validates :content, presence: true, length: {maximum: Settings.content_max_length}
+  default_scope->{order created_at: :desc}
```

Comment:
Em, used named scope instead of default_scope as it is rails anti-pattern. 
Reference: [http://weblog.jamisbuck.org/2015/9/19/default-scopes-anti-pattern.html](http://weblog.jamisbuck.org/2015/9/19/default-scopes-anti-pattern.html)